nextcloudappstore/core/fixtures/categories.json

```diff
@@ -23,120 +25,180 @@
         "fields": {
             "language_code": "fr",
             "name": "Outil",
+            "description": "",
             "master": "tools"
         }
     },
     {
         "model": "core.categorytranslation",
-        "pk": 4,
+        "pk": 10,
         "fields": {
             "language_code": "en",
-            "name": "Games",
-            "master": "games"
+            "name": "Multimedia",
+            "description": "Audio, video and picture apps",
+            "master": "multimedia"
         }
     },
     {
         "model": "core.categorytranslation",
-        "pk": 5,
+        "pk": 11,
         "fields": {
             "language_code": "de",
-            "name": "Spiele",
-            "master": "games"
+            "name": "Multimedia",
+            "description": "",
+            "master": "multimedia"
         }
     },
     {
         "model": "core.categorytranslation",
-        "pk": 6,
+        "pk": 12,
         "fields": {
             "language_code": "fr",
-            "name": "Jeux",
-            "master": "games"
+            "name": "Multimedia",
+            "description": "",
+            "master": "multimedia"
         }
     },
     {
         "model": "core.categorytranslation",
-        "pk": 7,
+        "pk": 13,
         "fields": {
             "language_code": "en",
-            "name": "PIM",
-            "master": "pim"
+            "name": "Files",
+            "description": "File mangement and Files app extension apps",
+            "master": "files"
         }
     },
     {
         "model": "core.categorytranslation",
-        "pk": 8,
+        "pk": 14,
         "fields": {
-            "language_code": "de",
-            "name": "PIM",
-            "master": "pim"
+            "language_code": "en",
+            "name": "Office & text",
+            "description": "Office and text processing apps",
+            "master": "office"
```

Comment:
names and descriptions are translated https://github.com/nextcloud/appstore/blob/master/nextcloudappstore/core/models.py#L147